sewer/cli.py

```diff
@@ -207,6 +207,12 @@ def main():
                 "ERROR:: Please supply {0} as an environment variable.".format(
                     str(e)))
             raise
+    elif dns_provider in ["aliyun", "alicloud"]:
+        from . import AliyunDNS
+        aliyun_ak = os.environ["ALIYUN_AK_ID"]
+        aliyun_secret = os.environ["ALIYUN_AK_SECRET"]
+        aliyun_endpoint = os.environ.get("ALIYUN_ENDPOINT", "cn-beijing")
+        dns_class = AliyunDNS(aliyun_ak, aliyun_secret, aliyun_endpoint)
```

Comment:
@menduo 
Hi, 
This should be in a try...except block so that if there is a failure getting the `ALIYUN_AK_ID` etc from environment; we can provide a helpful log message. 
for example see the way AuroraDns 
 is implemented here-> https://github.com/komuw/sewer/blob/b6482df6adfe7448a7940c355cf3c78cf0cf897d/sewer/cli.py#L175-L189

Reply:
OK, done.